shipyard_airflow/control/middleware.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2017 AT&T Intellectual Property.  All other rights reserved.
```

Comment:
Correct. We have consensus on using keystone for AAA in all UCP services. That pattern should hold of translating a authenticated token into a username and roles once Keystone is in place.